docs/user-manual/en/upgrading.md

```diff
@@ -16,18 +16,18 @@ Because of this separation it's very easy to upgrade Artemis in most cases.
 
 > **Note:**
 >
-> It's recommended to choose a folder different than the on where Apache
+> It's recommended to choose a folder different from where Apache
 > Artemis was downloaded. This separation allows you run multiple broker
 > instances with the same Artemis "home" for example. It also simplifies
 > updating to newer versions of Artemis. 
-
+ 
 ## General Upgrade Procedure
 
 Upgrading may require some specific steps noted in the [versions](versions.md),
 but the general process is as follows:
 
 1. Navigate to the `etc` folder of the broker instance that's being upgraded
-1. Open `artemis.profile` (`artemis.profile.cmd` on Windows). It contains a
+2. Open `artemis.profile` (`artemis.profile.cmd` on Windows). It contains a
```

Comment:
Having both lines begin with `1.` looks odd, but it doesn't necessarily need to change because the `1.` is just a placeholder to indicate a numbered list. When it's converted to HTML it will have the proper numbering.

Reply:
I did that because the steps were later wrong.

Reply:
I agree, its a separate 'set of tasks'. Besides, it looks to be a seperate list structurally in the file either way, in which case doesnt Markdown just ignore the actual number and do what it wants anyway?